src/control/cmd/dmg/system.go

```diff
@@ -111,47 +154,27 @@ func (cmd *systemStopCmd) Execute(args []string) error {
 	return nil
 }
 
-// systemMemberQueryCmd is the struct representing the command to shutdown system.
-type systemMemberQueryCmd struct {
+// systemStartCmd is the struct representing the command to start system.
+type systemStartCmd struct {
 	logCmd
 	connectedCmd
 }
 
-// Execute is run when systemMemberQueryCmd activates
-func (cmd *systemMemberQueryCmd) Execute(args []string) error {
-	members, err := cmd.conns.SystemMemberQuery()
-	if err != nil {
-		return errors.Wrap(err, "System-Member-Query command failed")
-	}
+// Execute is run when systemStartCmd activates
+func (cmd *systemStartCmd) Execute(args []string) error {
+	msg := "SUCCEEDED: "
 
-	cmd.log.Debug("System-Member-Query command succeeded\n")
-	if len(members) == 0 {
-		cmd.log.Info("No members in system\n")
-		return nil
-	}
-
-	rankTitle := "Rank"
-	uuidTitle := "UUID"
-	addrTitle := "Control Address"
-	stateTitle := "State"
-
-	formatter := txtfmt.NewTableFormatter(rankTitle, uuidTitle, addrTitle, stateTitle)
-	var table []txtfmt.TableRow
-
-	for _, m := range members {
-		row := txtfmt.TableRow{rankTitle: fmt.Sprintf("%d", m.Rank)}
-		row[uuidTitle] = m.UUID
-		row[addrTitle] = m.Addr.String()
-		row[stateTitle] = m.State().String()
-
-		table = append(table, row)
+	err := cmd.conns.SystemStart()
+	if err != nil {
+		msg = errors.WithMessagef(err, "FAILED").Error()
 	}
 
-	cmd.log.Info(formatter.Format(table))
+	cmd.log.Infof("System-start command %s\n", msg)
 
 	return nil
 }
 
+// Execute is run when systemListPoolsCmd activates
```

Comment:
Bad merge?

Reply:
will fix in https://github.com/daos-stack/daos/pull/1662